tools/debugging/replay_wal.py

```diff
@@ -81,9 +85,7 @@ def __getitem__(self, key):
             try:
                 return dict.__getitem__(self, alt)
             except KeyError:
-                import pdb
-
-                pdb.set_trace()  # pylint: disable=no-member
+                # breakpoint()
```

Comment:
Why the commented out code here?

Reply:
I'm not even sure if we still should have the breakpoint here. I believe it was an old debugging leftover/guard for uncovered edge cases.
I commented it out, because I wanted full commit-hook linting but did not want to make the call to remove it alltogether.